internal/service/federatedsettingsorgrolemapping/data_source_federated_settings_org_role_mapping.go

```diff
@@ -80,20 +79,20 @@ func dataSourceMongoDBAtlasFederatedSettingsOrganizationRoleMappingRead(ctx cont
 		return diag.FromErr(errors.New("role_mapping_id must be configured"))
 	}
 
-	federatedSettingsOrganizationRoleMapping, _, err := conn.FederatedSettings.GetRoleMapping(ctx, federationSettingsID.(string), orgID.(string), roleMappingID.(string))
+	federatedSettingsOrganizationRoleMapping, _, err := conn.FederatedAuthenticationApi.GetRoleMapping(ctx, federationSettingsID.(string), roleMappingID.(string), orgID.(string)).Execute()
```

Comment:
feel free to resolve after reading. I am seeing that orgID and roleMappingID are inverted, just making sure it's not undesired

Reply:
Correct, new signature:
`GetRoleMapping(ctx context.Context, federationSettingsId string, id string, orgId string) GetRoleMappingApiRequest`
[Github](https://github.com/mongodb/atlas-sdk-go/blob/main/admin/api_federated_authentication.go#L246)